modules/nextflow/src/test/groovy/nextflow/script/ScriptBindingTest.groovy

```diff
@@ -133,7 +134,7 @@ class ScriptBindingTest extends Specification {
 
         then:
         map['field1'] == 1
-        map['field-1'] == null
```

Comment:
this test case changes because camel case is a bit ambiguous with numbers- `field-1` normalizes to `field1` but field1 wasn't converting via `camelCaseToHyphen` into `field-1`.